JustSaying.UnitTests/JustSayingFluentlyTestBase.cs

```diff
@@ -1,8 +1,12 @@
 using System.Reflection;
+using System.Threading.Tasks;
 using JustSaying.AwsTools.QueueCreation;
 using JustBehave;
 using Microsoft.Extensions.Logging;
 using NSubstitute;
+using Xunit;
+
+[assembly: CollectionBehavior(DisableTestParallelization = true)]
```

Comment:
Can we put this in it's own `AssemblyInfo.cs` (or similar) file so it's easy to find?